apps/showcase/src/components/training/code-editor-view/code-editor-view.component.scss

```diff
@@ -26,3 +26,7 @@ ngx-monaco-editor {
 .editor-view {
   background: #1d1d1d;
 }
+
+.monaco-overflow-widgets {
+  z-index: 1022;
```

Comment:
magic number? :D

Reply:
Yeah... I could add a comment on that, I need it to be greater than the header which is `1021`

Reply:
updated with css var